include/podio/Frame.h

```diff
@@ -199,6 +199,29 @@ class Frame {
   ///          if it is not
   const podio::CollectionBase* get(const std::string& name) const;
 
+  /// Get the collection to which the passed object belongs from the Frame.
+  ///
+  /// @tparam CollT  The type of the desired collection
+  /// @param  object The object for which the collection it belongs to should
+  ///                be retrieved
+  ///
+  /// @returns       A const reference to the collection if it is available or to
+  ///                an empty (static) collection
+  template <CollectionType CollT>
+  const CollT& get(const CollT::value_type& object) const;
```

Comment:
```suggestion
  template <CollectionType CollT>
  const CollT& get(const typename CollT::value_type& object) const;
```
Adding a typename here (and in definition below) shouldn't hurt (although we already `value_type` to be type in the concept)

Reply:
Interesting that this seems to work without `typename` for gcc11, but not for gcc13. Fixed.